portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase

```diff
@@ -33,9 +33,51 @@ definition {
 	@ignore = "true"
 	@priority = "5"
 	test AnonymousIndividualsDoNotCountAsIndividualsForUsage {
+		ACUtils.addSiteAndPage();
 
-		// TODO LRAC-8147 Usage#AnonymousIndividualsDoNotCountAsIndividualsForUsage pending implementation
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
 
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "${propertyName}");
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUsage();
+
+		var currentIndividuals = selenium.getText("ACUsage#USAGE_CURRENT_INDIVIDUALS");
+
+		ACUtils.launchDXP();
+
+		JSONUser.addUser(
+			userEmailAddress = "userea@liferay.com",
+			userFirstName = "userfn",
+			userLastName = "userln",
+			userScreenName = "usersn");
+
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "userea@liferay.com",
+			userLoginFullName = "userfn userln");
+
+		Navigator.gotoSitePage(
+			pageName = "AC Page",
+			siteName = "Site Name");
+
+		ACUtils.closeAllSessions();
+
+		ACUtils.launchAC();
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUsage();
+
+		var newIndividuals = selenium.getText("ACUsage#USAGE_CURRENT_INDIVIDUALS");
+
+		if ("${currentIndividuals}" == "${newIndividuals}") {
```

Comment:
If I understand this if statement correctly, the assertion will only run if currentIndividuals = newIndividuals.

But this is a problem, since it will skip the assertion any time that currentIndividuals does not equal newIndividuals. Normally the test should fail in this case, but with the if statement it will just skip the failure.

We want to make sure the test fails properly if the behavior is not expected :)

Reply:
hey @AfonsoHSantos , yeah, you don't need to use the ``if`` clause, just use the assert, if the values are different the test will fail, if not the test will pass. :)

Reply:
Hm, make sense! fixed